refactorig(7.12 getting started tutorial) - updates to new 2021.1 release and improvements (#1377)
bonitasoft/bonita-doc: md/declare-contracts.md

# Declare contracts

> So far our process defines a sequences of events and tasks, and declares a business variable that it will instantiate and update when executed.  

Reviewer: ```suggestion
So far our process defines a sequence of events and tasks, and declares a business variable that it will instantiate and update when executed.  
```

> We want to instantiate our `claim` business variable using the description provided by the customer when they start the process. We will also need to update the claim with the answer provided by the employee and with the satisfaction rating provided by the customer. We need a way to collect information from the user and store it in the business variable. This is achieved using the user interfaces (web forms by default in Bonita) and based on another concept: [contract](contracts-and-contexts.md).

Reviewer: ```suggestion
We want to instantiate our `claim` business variable using the description provided by the customer when they start the process. We will also need to update the claim with the answer provided by the employee and with the satisfaction rating provided by the customer. We need a way to collect information from the user and store it in the business variable. This is achieved using the user interfaces (auto-generated forms) and based on another concept: [contract](contracts-and-contexts.md).
```

> A contract defines the data expected from the user to start a process (instantiation contract) or to execute a user task (task contract). By default Bonita offers a solution to easily build forms (_[see next chapter](create-web-user-interfaces.md)_) to allow the user to view and provide data. 

Reviewer: ```suggestion
A contract defines the data expected from the user (or an API call) to start a process (instantiation contract) or to execute a user task (task contract). By default, Bonita offers a solution to easily build forms (_[see next chapter](create-web-user-interfaces.md)_) to allow the user to view and provide data. 
```
